controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -332,27 +338,34 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					Name:      util.OcsOperatorConfigName,
 					Namespace: r.OperatorNamespace,
 				},
-			},
-			handler.EnqueueRequestsFromMapFunc(
-				func(context context.Context, obj client.Object) []reconcile.Request {
-					return []reconcile.Request{{
-						NamespacedName: InitNamespacedName(),
-					}}
-				},
-			),
+			}, enqueueOCSInit,
```

Comment:
Same as last comment
```suggestion
			}, 
			enqueueOCSInit,
```